testing/openapi_validation/validators/enum.go

```diff
@@ -57,6 +57,12 @@ var (
 			// vSphere
 			"apiVersions[0].openAPISpec.properties.cloud.properties.provider",
 		},
+		"global-hooks/openapi/values.yaml": {
+			"properties.clusterConfiguration.properties.apiVersion",
+			"properties.clusterConfiguration.properties.cloud.properties.provider",
+			"properties.deckhouseEdition",
```

Comment:
Let's change enum to `[Unknown, CE, FE, EE ]`. The linter will be satisfied, and this change seems to affect only one hook.

Reply:
done